web/client/actions/dashboards.js

```diff
@@ -14,7 +14,11 @@ const DASHBOARD_DELETED = "DASHBOARDS:DASHBOARD_DELETED";
 const RELOAD = "DASHBOARDS:RELOAD_DASHBOARDS";
 const LOADING = "DASHBOARDS:LOADING";
 
+const { success: successNotification } = require('./notifications');
+
+
 module.exports = {
+    showSuccessNotification: () => successNotification({ title: "success", message: "resources.successSaved" }),
```

Comment:
This is replicated 3 times in 3 files and then again in the files. 
This is not necessary. There is a little of duplication for these tools that have to be put in common. Please include these declarations in the plugins.